pkg/services/store/service.go

```diff
@@ -52,6 +54,9 @@ type CreateFolderCmd struct {
 type StorageService interface {
 	registry.BackgroundService
 
+	// Register the HTTP
+	RegisterHTTPRoutes(routing.RouteRegister)
```

Comment:
This PR removes the additional HTTP only service, and just adds a single function to register routes.  This lets us keep most of the details private, but still encapsulated in the `http.go` file